desc/objectives/_geometry.py

```diff
@@ -528,7 +534,9 @@ def build(self, eq, use_jit=True, verbose=1):
 
         self._dim_f = surface_grid.num_nodes
         self._data_keys = ["R", "phi", "Z"]
-        self._args = get_params(self._data_keys)
+        self._args = get_params(
+            self._data_keys, has_axis=plasma_grid.axis.size or surface_grid.axis.size
```

Comment:
FYI the surface grid should never have a point at the axis (since its a 2d grid over a surface) and realistically the plasma grid shouldn't either.